src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -0,0 +1,135 @@
+==================

```

Comment:
Here's what the new documentation format would look like

Reply:
Looks Great =)

Reply:
Suggestion: It might be nice to show boolean values using checkboxes. 
E.g. something like: https://stackoverflow.com/a/58639467

Reply:
@kennyweiss not sure about that. It is going to be a mixture of default values different for each type. checkbox might be confusing.

Reply:
Oh you mean for the "required" field... yea that might be nice actually.

Reply:
done!